modules/mono/csharp_script.cpp

```diff
@@ -374,6 +375,11 @@ String CSharpLanguage::validate_path(const String &p_path) const {
 	if (keywords.find(class_name)) {
 		return RTR("Class name can't be a reserved keyword");
 	}
+	Ref<TextServer> ts = TextServerManager::get_singleton()->get_primary_interface();
+	if(!ts->is_valid_identifier(class_name)) {
```

Comment:
We can use the macro instead.

```suggestion
	if (!TS->is_valid_identifier(class_name)) {
```